### PR TITLE
feat(public-cloud): enable k8s in US

### PR DESCRIPTION
**Pull request by radireddy**
ref: MANAGER-6599

Signed-off-by: Ravindra Adireddy <ravindra.adireddy@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-6599
| License          | BSD 3-Clause

## Description

Enable Kubernetes service in US

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/project.controller.js [hunk 1]
```diff
@@ -45,9 +46,31 @@ export default class ProjectController {
     this.projectQuotaAboveThreshold = this.quotas.find(
       (quota) => quota.quotaAboveThreshold,
     );
+
+    const featuresName = ProjectController.findFeatureToCheck(ACTIONS);
+    this.ovhFeatureFlipping
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/project.controller.js:51]: **nitpick:** Maybe we can add a `return` statement here.

Comment (radireddy) [reply packages/manager/modules/pci/src/projects/project/project.controller.js:51]: return statement in $onInit method? why and what are the advantages?

Comment (antleblanc) [reply packages/manager/modules/pci/src/projects/project/project.controller.js:51]: Not mandatory in this context here (`$onInit`), but usually when it comes to deal with Promises, it still a good habit to use a `return` statement. That's why here I'm adding this comment as a `nitpick`.

**Review by antleblanc — APPROVED** [review]

**Review by marie-j — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Comment by antleblanc** [comment]
Update target branch to `release/2021-w24`.

**Pull request merged**